cfgov/unprocessed/apps/tccp/css/main.less

```diff
@@ -116,6 +116,35 @@
   }
 }
 
+.m-apr-rating {
+  &--less .cf-icon-svg {
+    color: var(--green);
+  }
+
+  &--average .cf-icon-svg {
+    color: var(--gold);
+  }
+
+  &--more .cf-icon-svg {
+    color: var(--red);
+  }
+}
+
+.heading-4-size-only() {
+  @h4-font-size: @size-iv;
+
+  font-size: @h4-font-size;
```

Comment:
Do you need `@h4-font-size`? Could you just set `@size-iv` directly?

Reply:
I was trying to follow the way it's done in the DS, but happy to simplify.